introduction.md

```diff
@@ -61,6 +61,15 @@ To install Lean in your computer consider using the [Quickstart](https://github.
 
 This tutorial describes the current version of Lean, known as Lean 4.
 
+Be aware that some tactics in Lean's core library use classical reasoning by
```

Comment:
This definitely doesn't belong here; this is not a section about constructivity.

Reply:
If you want to remove the paragraph from here, I'll do it. But wouldn't constructivists appreciate it if this textbook informed them about the current behaviors of some tactics in Lean core before they begin to read the book?